src/content/operations/troubleshooting/_index.en.md

```diff
@@ -228,6 +228,8 @@ configmap submariner-lighthouse-coredns` and make sure it has following configur
     }
 ```
 
+In order to enable debug logs in Lighthouse CoreDNS pods, you can replace `errors` with `debug` in the above configmap.
```

Comment:
Should also add a note that it will log every DNS query which can fill the logs very quickly in an active system. So it is not recommended to set it to `debug` for normal operation or for long.

Reply:
> log every DNS query